osu.Game.Rulesets.Catch/Skinning/Legacy/CatchLegacySkinTransformer.cs

```diff
@@ -72,6 +72,12 @@ public override Drawable GetDrawableComponent(ISkinComponent component)
                         return new LegacyCatchComboCounter(Source);
 
                     break;
+
+                case CatchSkinComponents.HitExplosion:
+                    if (GetTexture("scoreboard-explosion-2") != null)
```

Comment:
This check should be based on the presence of a catcher sprite, not the explosion sprite directly. This particular texture is actually not skinnable in `osu-stable` (and we don't want to introduce no skinnable scope where it wasn't there previously) and should always use the default legacy skin implementation.